pkg/scheduler/queue/queue_test.go

```diff
@@ -93,156 +104,6 @@ func makeSchedulerRequest(tenantID string, additionalQueueDimensions []string) *
 	}
 }
 
-// TestMultiDimensionalQueueFairnessSlowConsumerEffects emulates a simplified queue slowdown scenario
-// which the scheduler's additional queue dimensions features are intended to solve for.
-//
-// In this scenario, one category of queue item causes the queue consumer to slow down, introducing a
-// significant delay while the queue consumer processes it and before the consumer can dequeue the next item.
-// This simulates a situation where one of the query components - the ingesters or store-gateways - is under load.
-//
-// If queue items belonging to the slow category are in the same queue ("normal-channel") in front of the normal queue
-// items, the normal queue items must wait for all slow queue items to be cleared before they can be serviced.
-// In this way, the degraded performance of the slow query component equally degrades the performance of the
-// queries which *could* be serviced quickly, but are waiting behind the slow queries in the queue.
-//
-// When using multiple queue dimensions, the queues are split by which "component" the query will utilize -- in this
-// test, those components are called "normal-channel" and "slow-channel" for clarity. The queue broker then
-// round-robins between the multiple queues, which has the effect of alternately dequeuing from the slow queries
-// and normal queries rather than blocking normal queries behind slow queries.
-func TestMultiDimensionalQueueFairnessSlowConsumerEffects(t *testing.T) {
```

Comment:
benchmark only relevant for comparing old versions of the tree and we have a better benchmark test now, introduced in `pkg/scheduler/queue/multi_queuing_algorithm_tree_queue_benchmark_test.go`